index.html

```diff
@@ -42,15 +54,220 @@
         </section>
       </section>
+
+      <section class="projects">
+        <section class="project">
+          <section class="project-image">
+            <img
+              src="./assets/images/project1.png"
+              alt="My first project snapshot"
+            />
+          </section>
+          <section class="project-name">Tonic</section>
+          <section class="project-year">
+            <span class="canopy">CANOPY</span>
+            <span class="dot"></span>
+            <span class="role">Back End Dev</span>
+            <span class="dot"></span>
+            <span class="role">2015</span>
+          </section>
+          <section class="project-description">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </section>
+          <section class="project-technologies">
+            <span>HTML</span>
+            <span>CSS</span>
+            <span>JavaScript</span>
+          </section>
+          <button type="button" class="project-button">See Project</button>
+        </section>
+      </section>
+
+      <section class="projects">
+        <section class="project">
+          <section class="project-image">
+            <img
+              src="./assets/images/project2.png"
+              alt="My first project snapshot"
+            />
+          </section>
+          <section class="project-name">Multi-Post Stories</section>
+          <section class="project-year">
+            <span class="canopy">CANOPY</span>
+            <span class="dot"></span>
+            <span class="role">Back End Dev</span>
+            <span class="dot"></span>
+            <span class="role">2015</span>
+          </section>
+          <section class="project-description">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </section>
+          <section class="project-technologies">
+            <span>HTML</span>
+            <span>CSS</span>
+            <span>JavaScript</span>
+          </section>
+          <button type="button" class="project-button">See Project</button>
+        </section>
+      </section>
+
+      <section class="projects">
+        <section class="project">
+          <section class="project-image">
+            <img
+              src="./assets/images/project3.png"
+              alt="My first project snapshot"
+            />
+          </section>
+          <section class="project-name">Tonic</section>
+          <section class="project-year">
+            <span class="canopy">CANOPY</span>
+            <span class="dot"></span>
+            <span class="role">Back End Dev</span>
+            <span class="dot"></span>
+            <span class="role">2015</span>
+          </section>
+          <section class="project-description">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </section>
```

Comment:
- [ ]  The paragraph (e.g., "A daily section ...") is a p tag, please work on it.